cc_corpus/download.py

```diff
@@ -67,34 +70,40 @@ def download_ranges(url: str,
             logging.error(f'Misc HTTP error for URL {url}: '
                           f'{r.status_code} - {r.text}; sleeping '
                           f'{orig_retry_left - retry_left}...')
-            time.sleep(orig_retry_left - retry_left)
+            time.sleep((orig_retry_left - retry_left) * delay_period)
             continue
     else:
         raise DownloadError(f'Could not download ranges from URL {url}.')
 
 
 def download_warc_ranges(
-    warc_file_name: str,
-    offsets_and_lengths: list[tuple[int, int]],
-    retry_left: int
+        warc_file_name: str,
+        offsets_and_lengths: list[tuple[int, int]],
+        retry_left: int,
+        delay: int = 10,
 ) -> list[bytes]:
     """
     Downloads byte ranges from a WARC file. A thin wrapper over
     :func:`download_ranges`.
     """
     return download_ranges(
         WARC_BASE_URI + warc_file_name,
-        offsets_and_lengths, retry_left
+        offsets_and_lengths, retry_left,
+        delay_period=delay
     )
 
 
 def download_index_range(
-    index_file_url: str,
-    offset: int,
-    length: int,
-    retry_left: int
+        index_file_url: str,
```

Comment:
Again, 4 spaces.

Reply:
Same thing, kept the 8 space indent, moved closing parenthes up one line.